src/components/Visualization/Visualization.js

```diff
@@ -212,25 +127,25 @@ export class UnconnectedVisualization extends Component {
             isLoading,
             onLoadingComplete,
         } = this.props
+
         const { renderId } = this.state
 
         return !visualization || error ? (
             <StartScreen />
```

Comment:
I'm considering removing the use of `VisualizationErrorInfo` from `StartScreen` and use it here directly when `error` is set.
I'm not 100% sure that `Visualization` is always rendered when there is an error set in Redux (`loader.loadingError`), if that is not the case then some errors won't be shown and most likely the app would crash.